aura_sr.py

```diff
@@ -748,16 +748,54 @@ def __init__(self, config: dict[str, Any], device: str = "cuda"):
         self.input_image_size = config["input_image_size"]
 
     @classmethod
-    def from_pretrained(cls, model_id: str = "fal-ai/AuraSR"):
+    def from_pretrained(cls, model_id: str = "fal-ai/AuraSR", use_safetensors: bool = True):
```

Comment:
should the default be use_safetensors=False if safetensors is not installed.;

Reply:
That's up to you! In HF libraries we recently made the shift to make `*.safetensors` default, so I was mirroring

Reply:
Having it error and force the person to add `use_safetensors=False` by hand is a way to make sure they really want to use pickle. But I guess not erroring, flipping to `use_safetensors=False` automatically and displaying a warning could be equally fine

Reply:
Or just make safetensors a dependency on `requirements.txt` and keep this PR as is. The strictness of safetensors is up to you. I'd recommend strongly nudge the users to use it.

Reply:
can you add it to setup.py?